css/style.css

```diff
@@ -0,0 +1,19 @@
+body{
```

Comment:
Consider putting a blank space between the word and the curly braces. I was one error that the linter is showing.